File: src/paths.js
const os = require('node:os');
const path = require('node:path');

const root = path.join(os.homedir(), '.claude-multi-cockpit');

module.exports = {
  root,
  profilesFile: path.join(root, 'profiles.json'),
  profilesDir: path.join(root, 'profiles'),
  profileDir(name) {
    return path.join(root, 'profiles', name);
  },
};

File: rust/src/paths.rs
use std::path::PathBuf;

pub fn root() -> PathBuf {
    dirs::home_dir()
        .expect("home directory not found")
        .join(".claude-multi-cockpit")
}

pub fn profiles_file() -> PathBuf {
    root().join("profiles.json")
}

pub fn profiles_dir() -> PathBuf {
    root().join("profiles")
}

pub fn profile_dir(name: &str) -> PathBuf {
    root().join("profiles").join(name)
}
